qcodes/__init__.py

```diff
@@ -73,7 +72,7 @@
 from qcodes.instrument.sweep_values import SweepFixedValues, SweepValues
 
 from qcodes.utils import validators
-from qcodes.utils.zmq_helpers import Publisher
```

Comment:
These changes should not be here.

Reply:
I was just commenting on that. Give me a minute :-)

Reply:
Sorry no rush :)